watertap/unit_models/zero_order/iron_and_manganese_removal_zo.py

```diff
@@ -0,0 +1,103 @@
+###############################################################################
+# WaterTAP Copyright (c) 2021, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National
+# Laboratory, National Renewable Energy Laboratory, and National Energy
+# Technology Laboratory (subject to receipt of any required approvals from
+# the U.S. Dept. of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#
+###############################################################################
+"""
+This module contains a zero-order representation of an iron and manganese removal unit
+"""
+
+from pyomo.environ import Constraint, units as pyunits, Var, Param
+from idaes.core import declare_process_block_class
+from watertap.core import build_sido, constant_intensity, ZeroOrderBaseData
+
+# Some more information about this module
+__author__ = "Chenyu Wang"
+
+
+@declare_process_block_class("IronManganeseRemovalZO")
+class IronManganeseRemovalZOData(ZeroOrderBaseData):
+    """
+    Zero-Order model for an iron and manganese removal unit operation.
+    """
+
+    CONFIG = ZeroOrderBaseData.CONFIG()
+
+    def build(self):
+        super().build()
+
+        self._tech_type = "iron_and_manganese_removal"
+
+        build_sido(self)
+
+        self.air_water_ratio = Var(self.flowsheet().time,
+                                units=pyunits.dimensionless,
+                                doc="ratio of air to water")
```

Comment:
We generally capitalise the first letter in the docs strings (normal sentence capitalization).